crates/erg_compiler/context/initialize/funcs.rs

```diff
@@ -21,75 +21,75 @@ impl Context {
         } else {
             Private
         };
-        let T = mono_q("T", instanceof(Type));
-        let U = mono_q("U", instanceof(Type));
-        let Path = mono_q_tp("Path", instanceof(Str));
-        let t_abs = nd_func(vec![kw("n", mono(NUM))], None, Nat);
+        let T = mono_q(TY_T, instanceof(Type));
+        let U = mono_q(TY_U, instanceof(Type));
+        let Path = mono_q_tp(PATH, instanceof(Str));
+        let t_abs = nd_func(vec![kw(KW_N, mono(NUM))], None, Nat);
         let t_all = func(
-            vec![kw("iterable", poly("Iterable", vec![ty_tp(Bool)]))],
```

Comment:
The first letter seems to be capitalized("iterable" to "Iterable")
Wonder why the error didn't happen before
@mtshiba

Reply:
Thanks, maybe I misplaced `KW_ITERABLE` to `Iterable` ones 🤔  I've fixed it and rerun CI.

Reply:
Certainly test is passed, but please have mtshiba look at it again.

Reply:
I think nothing is wrong.
`iterable` is a parameter name, and `Iterable` is a type name for it.